domino_data/auth.py

```diff
@@ -28,6 +49,14 @@ def get_headers(self) -> Dict[str, str]:
         if self.api_key:
             return {"X-Domino-Api-Key": self.api_key, **self.headers}
 
+        if self.token_url is not None:
+            try:
```

Comment:
the function signature says  """Get headers with both JWT and API Key.""". from the code doesn't look like we are retuning both?

Reply:
Domino gets confused with both, the docstring the class is more accurate (`A client that authenticates all requests with either the API Key or JWT.`)

Reply:
Will update the function docstring, thanks